src/Main/PlaneSliders.jsx

```diff
@@ -23,13 +30,15 @@ function PlaneSliders(props) {
   const visRefs = [xVisibility, yVisibility, zVisibility]
   const scrollRefs = [xScroll, yScroll, zScroll]
   const { slicingPlanes, viewMode } = state.context.main
-  const [slidersClass, setClass] = useState(0);
+  const [slidersClass, setClass] = useState(0)
 
   useEffect(() => {
-    window.addEventListener("resize", () => getClass())
+    window.addEventListener('resize', () => getClass())
```

Comment:
There is a memory leak here.  Need to unregistered the addEventListener callback.  Does not matter as this component is long lived (for now), but it hurts my heart. 
```
useEffect(() => {
   const onResize = () => getClass()
    window.addEventListener('resize', onResize)
    return () => window.removeEventListener(onResize)
  }, []) 
```